kitchen.yml

```
@@ -8,32 +8,25 @@ driver:
 provisioner:
   name: chef_infra
   product_name: <%= ENV['CHEF_PRODUCT_NAME'] || 'chef' %>
-  product_version: <%= ENV['CHEF_VERSION'] || '17' %>
+  product_version: <%= ENV['CHEF_VERSION'] || '18' %>
```

shouldn't this just be latest for future proofing.
```suggestion
  product_version: <%= ENV['CHEF_VERSION'] || 'latest' %>
```